src/components/ui/icons/IconCompass.vue

```diff
@@ -1,14 +1,14 @@
 <template>
   <svg
-    width="35"
-    height="35"
+    width="18"
```

Comment:
I move the icon compass to the common UI last week, https://github.com/sigrennesmetropole/cooperation_jn_common_ui/pull/15

and move the usage on trambus here: https://github.com/sigrennesmetropole/cooperation_jn_trambus/pull/59